actioncable/lib/action_cable/subscription_adapter/naming.rb

```diff
@@ -0,0 +1,12 @@
+module ActionCable
+  module SubscriptionAdapter
+    module Naming
```

Comment:
`:nodoc:`

How about we call this `ChannelPrefix` or similar? SubscriptionAdapter::Naming feels a bit too vague to me.